package.json

```
@@ -1,13 +1,13 @@
 {
-  "name": "sd-0x-week11-project-jest",
+  "name": "sd-0x-project-jest",
   "version": "1.0.0",
-  "description": "Project for block 11",
+  "description": "Jest project",
   "scripts": {
     "test": "jest"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/betrybe/sd-0x-week11-project-jest.git"
+    aí
```

Isso não deveria continuar como era? :thinking:

```suggestion
    "url": "git+https://github.com/betrybe/sd-0x-week11-project-jest.git"
```